dot/peerset/peerstate.go

```diff
@@ -258,17 +259,19 @@ func (ps *PeersState) addNoSlotNode(idx int, peerID peer.ID) {
 	}
 
 	ps.nodes[peerID] = n
+	return nil
 }
 
-func (ps *PeersState) removeNoSlotNode(idx int, peerID peer.ID) {
+func (ps *PeersState) removeNoSlotNode(idx int, peerID peer.ID) error {
 	if _, ok := ps.sets[idx].noSlotNodes[peerID]; !ok {
-		return
+		logger.Debugf("peer %s is not in \"no slot node\" map", peerID)
```

Comment:
nit use back ticks it looks PrEtTiEr

```suggestion
		logger.Debugf(`peer %s is not in "no slot node" map`, peerID)
```

Reply:
addressed